### PR TITLE
copr: test_from_local_time update test case 

### LINKED ISSUE #12093: copr: test_from_local_time update test case
## Bug Report
codec::mysql::time::tests::test_from_local_time
test failed.

### What did you expect?
test passed.

### What did happened?
https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/17038/pipeline/116

### PR DESCRIPTION
**Pull request by xiejiandong**
Issue Number: Close #12093

What's Changed:
code:
components/tidb_query_datatype/src/codec/mysql/time/mod.rs
test_from_local_time() update test case, avoid time shift and test failed;

Tests
- Unit test
- Manual test

```release-note
None
```

### CODE REVIEW

**Review by BusyJay — APPROVED** [review]

File: components/tidb_query_datatype/src/codec/mysql/time/mod.rs [hunk 1]
```diff
@@ -2728,12 +2728,9 @@ mod tests {
...
 
-            //2022-02-15 07:45:48.581699 NonFixed(UTC)   c_datetime
-            //2022-02-15 07:45:48.581933075 UTC          Utc::now()
-            //We compare length 22 of then; Exceed it may be not equal ( in milli seconds ).
-            let now0 = &c_datetime.to_string()[0..22];
-            let now1 = &Utc::now().to_string()[0..22];
-            assert_eq!(now0, now1);
+            let now0 = c_datetime.timestamp_millis() as u64;
+            let now1 = Utc::now().timestamp_millis() as u64;
+            assert_eq!(now1 - now0 < 1000, true);
```
Comment (BusyJay) [inline components/tidb_query_datatype/src/codec/mysql/time/mod.rs:2733]: Should be `assert!(now1 - now0 < 1000, "{:?} {:?}", now1, now0)`.

Comment (xiejiandong) [reply components/tidb_query_datatype/src/codec/mysql/time/mod.rs:2733]: ok

**Review by skyzh — APPROVED** [review]

**Pull request merged**